modules/rosa-policy-incident.adoc

```diff
@@ -109,6 +109,13 @@ When managing a new incident, Red Hat uses the following general workflow:
 . The incident is documented and a root cause analysis (RCA) is performed within 5 business days of the incident.
 . An RCA draft document will be shared with the customer within 7 business days of the incident.
 
+Red Hat is also available to assist with customer incidents when raised through a support case. 
+Red Hat is able to assist with activities including but not limited to:
```

Comment:
```suggestion
Red Hat assists with activities including but not limited to:
```

Reply:
What about 

```suggestion
Red Hat can assist with activities including but not limited to:
```

I'm just trying to make it more obvious that we're not providing this assistance automatically or proactively, but that we _can_ provide this assistance when requested

Reply:
I think the "can" works nicely.